src/db/delivery_problems/test.rs

```diff
@@ -113,15 +113,15 @@ fn check_soft_bounce() {
     assert_eq!(error.code(), 429);
     assert_eq!(error.errno().unwrap(), 107);
     assert_eq!(error.error(), "Too Many Requests");
-    assert_eq!(error.to_string(), "Email account soft bounced");
+    assert_eq!(error.to_string(), "Soft-bounce limit violated");
```

Comment:
I tweaked these error messages to make it clearer that the error is not saying an email necessarily bounced just now, but that the configured limits have been violated (so the bounce itself may have been some time ago).